.github/workflows/main.yml

```diff
@@ -65,8 +65,12 @@ jobs:
           PYINSTALLER_COMPILE_BOOTLOADER: 1
 
       - name: Prep kolibri dist
+        env:
+          KOLIBRI_WHL_URL: 'https://buildkite.com/organizations/learningequality/pipelines/kolibri-python-package/builds/6389/jobs/018385cc-d517-4f48-b281-193b9933d562/artifacts/018385d4-10d6-443b-868e-319b7c4eb2de'
+          KOLIBRI_WHL_FILENAME: "kolibri-0.16.0.dev0+git.20220928203123-py2.py3-none-any.whl"
         run: |
-          pip install kolibri --target="src"
+          C:\msys64\usr\bin\wget.exe -O $env:KOLIBRI_WHL_FILENAME $env:KOLIBRI_WHL_URL
```

Comment:
We can try to use the action [engineerd/configurator](https://github.com/engineerd/configurator) to download the file in the future.